spec/controllers/verify/finance_controller_spec.rb

```diff
@@ -84,7 +84,18 @@
 
     context 'when form is valid' do
       context 'when CCN is confirmed' do
+        it 'redirects to the verify address page, if feature is enabled' do
```

Comment:
I don't think we need to change anything in this spec since we didn't change the controller. It's expected to always redirect to `verify/address`, whether the feature is enabled or not.